Update 8/15/18

wheikoff opened the pull request.

kbuzard commented on Will Heikoff Documentation:
```diff
@@ -320,4 +320,64 @@ The assumption that the Zip Code Business Pattern Data is based off of USPS Zip
+Updated ZIPs_Tot_Emp.R file to extract only California Zip Codes from the file
+2000 ZCTA Manufa. Data has 198 Zips with no Manufa. Employment, 2000 ZBP Manufa. Data has 0 Zips with no Manufa. Employment, 1998 ZBP Manufa. Data has 0 Zips with no Manufa. Employment
+Earliest ZCTA Shapefiles available are 2007
+Downloaded 2000 ZCTA Cartographic Boundary Files for California ftp://ftp2.census.gov/geo/tiger/PREVGENZ/zt/z500shp/, titled zt06_d00.shp, saved on the desktop
```
We have something better than this, right? So we can delete this and make a note that we didn't end up using it?

wheikoff commented:
Yes, sorry I forgot to take that out of my documentation.

On Aug 16, 2018, at 12:29 PM, Kristy Buzard <notifications@github.com<mailto:notifications@github.com>> wrote:


@kbuzard commented on this pull request.

________________________________

In Will Heikoff Documentation<https://github.com/kbuzard/labs/pull/20#discussion_r210697297>:

> +Working with K-function Local_3Stage.py program - Entered “G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\Complete Zip Code Industry Detail File\CAL_ZCTA_EMP_2.shp” into line 37 to set the zips file
+Entered “Number__Em” and “Id2” onto line 40 to set the zip features
+Entered “Number__Em” on line 44 to set the Reference Measure
+Entered “Id2” on line 47 to set the ID Field
+Running program with new Zip Code File for two simulations
+Simulations completed in 3.35 minutes
+Received same “GDAL data files not located, GDAL_DATA not set” and “PROJ data files not located, PROJ_LIB not set” errors, and …_buffers.shp file not created
+Trying to set GDAL by entering “conda create -n GDAL python=3.6 gdal -c conda-forge” into Anaconda Prompt
+       Installation successful; after running the program again, the errors still occur and still no …_buffers.shp built
+
+8/6/18
+Adjusted CA_WAC_Transformation.R file to format the Block ID to add the necessary 0 at the beginning of the ID directly in R rather than having to do it in ArcMap
+Updated ZIPs_Tot_Emp.R file to extract only California Zip Codes from the file
+2000 ZCTA Manufa. Data has 198 Zips with no Manufa. Employment, 2000 ZBP Manufa. Data has 0 Zips with no Manufa. Employment, 1998 ZBP Manufa. Data has 0 Zips with no Manufa. Employment
+Earliest ZCTA Shapefiles available are 2007
+Downloaded 2000 ZCTA Cartographic Boundary Files for California ftp://ftp2.census.gov/geo/tiger/PREVGENZ/zt/z500shp/, titled zt06_d00.shp, saved on the desktop


We have something better than this, right? So we can delete this and make a note that we didn't end up using it?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/kbuzard/labs/pull/20#pullrequestreview-146969088>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AmkU6qqoNSxADHcRyoCKf7txA86WH9Btks5uRboAgaJpZM4V-8KR>.
